### PR TITLE
use of ctx.state

### PR DESCRIPTION
**Pull request by fundon**
#5

### CODE REVIEW

File: lib/csrf.js [hunk 1]
```diff
@@ -52,10 +52,7 @@ module.exports = function (options) {
     var validate = impl.validate || _impl.validate;
     var _token = _impl.token || _impl;
     // Set the token
-    if (!this.locals) {
```
Comment (fengmk2) [inline lib/csrf.js]: How about make this compatible?

Comment (fengmk2) [inline lib/csrf.js]: Oh, I never use this csrf, never mind.

Comment (fengmk2) [inline lib/csrf.js]: .state start from which version of koa?

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
release patch version: 2.0.1

File: lib/csrf.js (hunk 1 above)
Comment (fundon) [inline lib/csrf.js]: start from **v0.14**.

**Pull request merged**